tests/baselines/reference/objectTypeLiteralSyntax2.errors.txt

```diff
@@ -15,4 +17,8 @@ tests/cases/conformance/types/objectTypeLiteral/objectTypeLiteralSyntax2.ts(12,2
     
     var z: { foo: string bar: string }
                          ~~~
+!!! error TS1005: '=' expected.
+                         ~~~
+!!! error TS2304: Cannot find name 'bar'.
+                            ~

```

Comment:
It should be sufficient if you make it so that you only try to parse an initializer if you see a `=`.